src/kdump-tools/patch/0003-Panic-on-kdump-kernel.patch

```diff
@@ -0,0 +1,20 @@
+From: Olivier Singla <olivier.singla@broadcom.com>
+Date: Mon Dec  2 15:25:46 EST 2019
+Subject: kdump: add "panic=X" option so system reboots if the capture kernel crashes or stay stuck
+
+In case the capture kernel (the kdump kernel started with kexec) would
+either crash or be stuck, the system should reboot. We need then to add
+the "panic=X" option to the kernel. Without this option, the system could
+stuck and not reboot.
+
+--- a/debian/kdump-tools.default.orig
++++ b/debian/kdump-tools.default
+@@ -71,7 +71,7 @@ KDUMP_COREDIR="/var/crash"
+ #     "irqpoll nr_cpus=1 nousb systemd.unit=kdump-tools.service ata_piix.prefer_ms_hyperv=0"
+ #KDUMP_KEXEC_ARGS=""
+ #KDUMP_CMDLINE=""
+-#KDUMP_CMDLINE_APPEND="irqpoll nr_cpus=1 nousb systemd.unit=kdump-tools.service ata_piix.prefer_ms_hyperv=0"
++KDUMP_CMDLINE_APPEND="irqpoll nr_cpus=1 nousb systemd.unit=kdump-tools.service ata_piix.prefer_ms_hyperv=0 panic=10 debug hpet=disable pcie_port=compat pci=nommconf platform=__PLATFORM__"
```

Comment:
can you explain why pass PLATFORM=$platform to the kernel cmdline? what is the use case here? I do not see this in the description.

Reply:
The PLATFORM string will be accessible to the reboot script. When the crash kernel is rebooting it requires $platform value to use any platform specific reboot script which is defined in /usr/share/sonic/device/$platform/platform_reboot.

Reply:
platform seems like a common keyword, do you know if kernel does not use it as a parameter already. will there be a chance of conflict? it would be better if you can put sonic_platform to avoid such potential conflict